p/PerlPlus/PerlPlus-5.34.1-GCCcore-11.3.0-v22.11.1.eb

```diff
@@ -26,7 +26,7 @@ dependencies = [
     ('LibTIFF', '4.3.0'),
     ('ncurses', '6.3'),
     ('R', '4.2.2', '-bare', ('foss', '2022a')),
-    ('MariaDB', '10.9.3', '', ('GCC', '11.3.0')),
+    ('MariaDB', '10.10.2', '', ('GCC', '11.3.0')),
```

Comment:
kunnen we dat zo updaten in PerlPlus? Of moeten we een nieuwe versie maken? @pneerincx @marieke-bijlsma

Reply:
Technisch gezien zou je voor de zekerheid een nieuwe `PerlPlus-5.34.1-GCCcore-11.3.0-v23.04.1.eb` moeten maken. Ik kan me nog de nodige terechte kritiek op andere partijen herinneren, die spullen updaten zonder versie nummers te wijzigen ;)